index.html

```diff
@@ -18,6 +18,11 @@
         <div class="logo">
           <a href="#"><img src="assets/images/mobile/My-Logo.svg" alt="Logo"></a>
         </div>
+        <ul class="nav-item">
+          <li class="nav-item1"><a href="#portfolio">Portfolio</a></li>
+          <li class="nav-item2"><a href="#about">About</a></li>
+          <li class="nav-item3"><a href="#contact">Contact</a></li>
+        </ul>
         <div class="menu-icon">
           <a href="#"><img src="assets/images/mobile/Icon-Menu.svg" alt="Menue"></a>
         </div>
```

Comment:
- [ ] As previously requested, for desktop version, the navbar should change and not look like the mobile version navbar.
![Screenshot from 2021-10-02 10-21-56](https://user-images.githubusercontent.com/65501535/135707554-33caaec7-17bf-4dd8-b968-6ababf9c7ff4.png)